packages/webapp/src/app/podcasts/[podcast_id]/page.tsx

```diff
@@ -0,0 +1,11 @@
+import { SearchBox } from '@/components/searchbox';
+
+export default function PodcastPage({ params }: { params: { podcast_id: string } }) {
+  return (
+    <div className="w-full max-w-5xl mt-8">
+      <div className="w-full flex justify-center mt-8">
+        <SearchBox podcast_id={params.podcast_id} endpoint={'../api/podcasts/query'} />
```

Comment:
Gotta provide a `corpusId` from KV here. Potentially we can have icons for the podcasts too.